docs/onboarding.rst

```diff
@@ -328,3 +328,8 @@ What is MS/PFS?
 How are messages encoded?
   The `messages specification <https://raiden-network-specification.readthedocs.io/en/latest/messaging.html>`_ should describe the message formats.  On the implementation side, the `messages file <https://github.com/raiden-network/raiden/blob/4979c27f5d031dd44963f7f7a4b10a7d20cee712/raiden/messages.py>`_ contains the details.  See ``encode()`` method of ``Message`` class, and ``packed()`` method of each subclass of ``Message``.  Ultimately the byte sequence of the packed messages is defined in the `encoding/messages file <https://github.com/raiden-network/raiden/blob/a19a6c853b55f13725f2545c77b0475cbcc86807/raiden/encoding/messages.py#L58>`_.
 
+Why does `updateNonClosingBalanceProof() <https://github.com/raiden-network/raiden-contracts/blob/04fe5a6c33b2a34cd893c5e546b8df949e194947/raiden_contracts/contracts/TokenNetwork.sol#L536>`_ require two signatures?
+  The closing participant's signature is required because the balance proof here is sent from the closing participant to the non-closing participant. On this direction of the channel, the closing participant has control over issuing balance proofs. On the other side, the non-closing participant has a choice of accepting or refusing the balance proofs. For example, when the transferred amount decreases in a new balance proof, the non-closing participant should refuse the balance proof. In such cases, the non-closing participant should never add the second signature on the balance proof.
+
+Why are balance hashes used in `closeChannel() <https://github.com/raiden-network/raiden-contracts/blob/04fe5a6c33b2a34cd893c5e546b8df949e194947/raiden_contracts/contracts/TokenNetwork.sol#L470>`_ and `updateNonClosingBalanceProof() <https://github.com/raiden-network/raiden-contracts/blob/04fe5a6c33b2a34cd893c5e546b8df949e194947/raiden_contracts/contracts/TokenNetwork.sol#L536>`_ instead of plain balance proofs?
+  The purpose is to delay the moment when the actual token amounts become visible to the public. Especially, Monitoring Services never need to see concrete token amounts.
```

Comment:
It also hides all intermediate transfer amounts from the public. So instead of knowing a user sent three transfers with the amounts 1, 1 and 20, you only know that he sent three transfers with a total of 22. Not sure if that has to go into this answer, but it does more than just delaying information release.

Reply:
I'll rephrase this paragraph.